ga4gh/frontend.py

```diff
@@ -12,13 +12,31 @@
 import flask.ext.api as api
 import flask.ext.cors as cors
 
+from flask.ext.api.decorators import set_renderers
+from flask.ext.api.renderers import HTMLRenderer
+
 app = api.FlaskAPI(__name__)
 app.config.from_object('ga4gh.server.config:DefaultConfig')
 if os.environ.get('GA4GH_CONFIGURATION') is not None:
     app.config.from_envvar('GA4GH_CONFIGURATION')
 
+app.config['DEFAULT_RENDERERS'] = [
+    'flask.ext.api.renderers.JSONRenderer',
+    'flask.ext.api.renderers.HTMLRenderer',
+    'flask.ext.api.renderers.BrowsableAPIRenderer',
+]
```

Comment:
I needed these changes to get the homepage to render, but I fear this may break the BrowsableAPIRenderer.  I tried using [@set_renderers(HTMLRenderer)](http://www.flaskapi.org/api-guide/renderers/#htmlrenderer), but am getting TypeError: 'type' object is not iterable